docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<ul><li>"Circular domains are <strong>a</strong> challenge for Gaussian Processes. Periodic patterns are assumed, but they are hard to capture with primitives <strong>and combining them is computationally expensive. An a</strong>lternative approach is required."</li><li>Also, could be interesting to add a few words about <em>why</em> circular domains are a challenge for GPs?</li><li>"<strong>the</strong> Weinland function is used to solve the problem and ensure<s>s</s> positive definite kernel<strong>s</strong> on the circular domain"</li><li>I suppose <code>t</code> is time? Would be interesting to add this precision, as this would also enlighten us as to the concrete meaning of <code>c</code>. Also interesting to explain rapidly what <code>tau</code> represents</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<ul><li>I don't understand this sentence: "For circular domain [0, 2π), how to model correlation between π−ε and ε, real distance is 2ε, but computes differently with simple math"</li><li>"For correctly computed distances we need to <s>ferify</s> <strong>verify the</strong> kernel we obtain is positive definite"</li><li>Also, that'd be nice if you have some time to address the last point of my previous comment ;) </li></ul>

Reply:
<p>updated</p>